sql/npc_list.sql

```diff
@@ -13894,16 +13894,16 @@ INSERT INTO `npc_list` VALUES (17232299,'Mogball-Local','Mogball-Local',0,0.000,
 INSERT INTO `npc_list` VALUES (17232300,'Mog-Tablet','Mog-Tablet',0,0.000,0.000,0.000,0,40,40,0,0,0,6,3,0x0000320000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17232301,'DebugMaster','DebugMaster',67,-23.446,-59.372,-77.446,1,40,40,0,0,0,2,27,0x0000520000000000000000000000000000000000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17232302,'Stooge_Moogle','Stooge Moogle',33,-161.092,-79.142,177.157,0,40,40,0,1,0,6,27,0x00003C0900000000000000000000000000000000,32,NULL,1);
-INSERT INTO `npc_list` VALUES (17232303,'Lonely_Evergreen','Lonely Evergreen',217,-161.824,-80.075,177.869,32769,40,40,0,17,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,1);
+INSERT INTO `npc_list` VALUES (17232303,'Lonely_Evergreen','Lonely Evergreen',217,-161.824,-80.075,177.869,32769,40,40,0,17,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',1);
```

Comment:
I think Lonely Evergreen, Pips, and Options are all `AMK`?

Reply:
Good catch! ty